tests/unit_tests/cpu/material/stopping_power.cpp

```diff
@@ -0,0 +1,147 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2024 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Project include(s).
+#include "detray/materials/interaction.hpp"
+#include "detray/materials/material.hpp"
+#include "detray/materials/predefined_materials.hpp"
+
+// Detray test include(s)
+#include "detray/test/utils/types.hpp"
+
+// GTest include(s).
+#include <gtest/gtest.h>
+
+using namespace detray;
+
+// Test class for the stopping power
+// Input tuple: < material, particle type, kinetic energy, expected output >
+class StoppingPowerValidation
+    : public ::testing::TestWithParam<
+          std::tuple<material<scalar>, pdg_particle<scalar>, scalar, scalar>> {
+};
+
+TEST_P(StoppingPowerValidation, stopping_power) {
+
+    // Interaction object
+    interaction<test::scalar> I;
+
+    // Material
+    material<scalar> mat = std::get<0>(GetParam());
+
+    // Particle
+    pdg_particle<scalar> ptc = std::get<1>(GetParam());
```

Comment:
Can you please use ```test::scalar``` everywhere? The global ```scalar``` type should not be used anymore

Reply:
OK. I thought it is just about that specific line

Reply:
Yes, sorry, my comment was not particularly clear

Reply:
It is fixed now